### PR TITLE
Handle possible errors which are not critical

### PR DESCRIPTION
**Pull request by sebrandon1**
Follow up to #934 

The script was [hitting an error](https://github.com/test-network-function/cnf-certification-test/actions/runs/4474518500/jobs/7863067358) with `oc adm` that was causing it to crash out prior to completion.  We handle the blank string so there's no problem with the error and continuing.

I re-ran the script to bring in the latest values with this PR.

### CODE REVIEW

File: script/rhcos_versions.sh [hunk 1]
```diff
@@ -1,7 +1,6 @@
 #!/usr/bin/env bash
 
 set -x
-set -e
```
Comment (rdavid) [inline script/rhcos_versions.sh:4]: Leave it, just mask the failure:
```
RHCOSVERSION="$(oc adm release info "${VERSION}" -o 'jsonpath={.displayVersions.machine-os.Version}') 2>&1" ||
  printf 'Continue with the error %s.\n' "$RHCOSVERSION"
```

Comment (rdavid) [reply script/rhcos_versions.sh:4]: Do you mean the script exits anyway?

Comment (sebrandon1) [reply script/rhcos_versions.sh:4]: No I copied and pasted your suggestion into my code and it doesn't run the way it should.

Comment (rdavid) [reply script/rhcos_versions.sh:4]: Try now, show me the output.

Comment (sebrandon1) [reply script/rhcos_versions.sh]: Edits committed.  Thanks!

**Pull request merged**